Problem5.js

```diff
@@ -0,0 +1,15 @@
+// Write a JavaScript program to compute the sum of three elements of a given array of integers of length 3.
+
+// let arr = [3,8,23]
+// sum = 0;
+// for(let i=0; i<arr.length; i++)
+// {
+//     sum = sum + arr[i]
+// }
+// console.log(sum)
+
+function arr(sum){
+   return sum[0]+sum[1]+sum[2]
```

Comment:
It should not be hard coded. Can you make this dynamic?
Program should handle any array of any length